egs/wsj/s5/steps/dict/prons_to_lexicon.py

```diff
@@ -0,0 +1,191 @@
+#!/usr/bin/env python
+
+# Copyright 2016  Vimal Manohar
+#           2016  Xiaohui Zhang
+# Apache 2.0.
+
+# we're using python 3.x style print but want it to work in python 2.x,
+from __future__ import print_function
+import argparse
+import sys
+
+class StrToBoolAction(argparse.Action):
+    """ A custom action to convert bools from shell format i.e., true/false
+        to python format i.e., True/False """
+    def __call__(self, parser, namespace, values, option_string=None):
+        if values == "true":
+            setattr(namespace, self.dest, True)
+        elif values == "false":
+            setattr(namespace, self.dest, False)
+        else:
+            raise Exception("Unknown value {0} for --{1}".format(values, self.dest))
+
+def GetArgs():
```

Comment:
We are moving the python to a pep8-compliant style (e.g. lower_case function names), and validation with flake8, but you could do that later.